configs/config.yaml.example

```diff
@@ -7,3 +7,6 @@ database:
 logging:
   level: debug
   console: True
+certs:
+  capath: "/cdncert/ca.pem"
```

Comment:
i think it might make sense to just embed the ca cert into our project.  Since its not a secret, i think this makes more sense.  Also, we would need to pull in the latest ca:  https://github.com/Katello/katello/blob/master/ca/redhat-uep.pem